web/src/components/Storyblok/Blockquote/Blockquote.module.scss

```diff
@@ -0,0 +1,59 @@
+@use '@nice-digital/nds-core/scss/typography';
+@use '@nice-digital/nds-core/scss/spacing';
+@use '@nice-digital/nds-core/scss/utils';
+@use '@nice-digital/nds-core/scss/glyphs';
+@use '@nice-digital/nds-core/scss/colours';
+
+////
+/// @group Typography
+////
+
+/// Blockquote style
+/// @since 0.1.0
```

Comment:
At some point we should probably decide how/if we want to implement sassdoc (items like `@since`, `@group` etc). I don't think it's been consistently applied so far (by myself too), so let's catch up about what we want to do here when things are less manic